src/modules/shared/hooks/use-sticky-layout/use-sticky-layout.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { setIsStickyLayout } from '@shared/store/ui';
+
+import { UseStickyLayout } from './use-stikcy-layout.types';
+
+const useStickyLayout: UseStickyLayout = (isSticky) => {
+	const dispatch = useDispatch();
+
+	useEffect(() => {
+		dispatch(setIsStickyLayout(isSticky));
+		return () => {
+			dispatch(setIsStickyLayout(false));
+		};
+	});
```

Comment:
Hier mist er nog een dependency array, ik zou die `isSticky` param ook by default op `true` zetten aangezien je dat meestal wel gaat willen bij het gebruiken van deze hook, die moet dan mee in de dep array.